src/software/field_tests/movement_robot_field_test.py

```diff
@@ -150,13 +150,18 @@ def test_one_robots_square(field_test_runner):
         ]
     )
 
+
+    # field_test_runner.gamecontroller.send_gc_command(
+    #     gc_command=Command.Type.FORCE_START, team=Team.BLUE
+    # )
+
     id = world.friendly_team.team_robots[0].id
     print(f"Running test on robot {id}")
 
-    point1 = Point(x_meters=-0.3, y_meters=0.6)
-    point2 = Point(x_meters=-0.3, y_meters=-0.6)
-    point3 = Point(x_meters=-1.5, y_meters=-0.6)
-    point4 = Point(x_meters=-1.5, y_meters=0.6)
+    point1 = Point(x_meters=-0.75, y_meters=0.6)
+    point2 = Point(x_meters=-0.75, y_meters=-0.6)
+    point3 = Point(x_meters=0.75, y_meters=-0.6)
+    point4 = Point(x_meters=0.75, y_meters=0.6)
```

Comment:
you should revert after you're done testing

Reply:
yep. We can remove this.